process_report/tests/util.py

```diff
@@ -80,3 +77,19 @@ def new_validate_pi_alias_processor(
 
 def new_lenovo_processor(name="", invoice_month="0000-00", data=pandas.DataFrame()):
     return lenovo_processor.LenovoProcessor(name, invoice_month, data)
+
+
+def new_validate_billable_pi_processor(
+    name="",
+    invoice_month="0000-00",
+    data=pandas.DataFrame(),
+    nonbillable_pis=[],
+    nonbillable_projects=[],
```

Comment:
don't pass mutables as defaults, set it to `None` by default. This would apply to data, nonbillable_pis and nonbillable_projects

Reply:
After you told me, I realize now why this is a big problem. All of the functions in `tests/util.py` has this problem, so I believe it should be something I'll cover in a bug fix PR? Could I attach this problem to issue #111?

Reply:
please do it as a separate issue and PR. However, this specific instance should be fixed it as part of this PR because it's introduced here